x/perpetual/keeper/mtp.go

```diff
@@ -163,12 +163,19 @@ func (k Keeper) fillMTPData(ctx sdk.Context, mtp types.MTP, ammPoolId *uint64, r
 	pnl := math.ZeroInt()
 	liquidationPrice := sdk.ZeroDec()
 	if realTime {
```

Comment:
Is it possible to remove this realTime ? This function just fills it never sets in KV store

Reply:
yes